app/serializers/costs_parameters_serializer.rb

```diff
@@ -5,46 +5,135 @@
+    costs_carriers_biomass: %w[
+      costs_carriers_biomass
     ],
-    costs_infrastructure_heat: %w[
-      costs_infrastructure_heat
+    costs_carriers_oil_and_products: %w[
+      costs_carriers_oil_and_products
     ],
-    costs_infrastructure_hydrogen: %w[
-      
+    costs_carriers_coal_and_products: %w[
+      costs_carriers_coal_and_products
     ],
-    costs_infrastructure_network_gas: %w[
-      gas_network_total_costs
+    costs_carriers_uranium: %w[
+      costs_carriers_uranium
+    ],
+    costs_carriers_heat: %w[
+      costs_carriers_heat
+    ],
+    costs_carriers_natural_gas: %w[
+      costs_carriers_natural_gas
+    ],
+    costs_carriers_waste: %w[
+      costs_carriers_waste
     ],
-    costs_co2_molecule_nodes: %w[
-      costs_co2_energy_graph
+    costs_carriers_electricity: %w[
+      costs_carriers_electricity
+    ],
+    costs_carriers_hydrogen: %w[
+      costs_carriers_hydrogen
+    ],
+    costs_carriers_liquid_hydrogen: %w[
+      costs_carriers_liquid_hydrogen
+    ],
+    costs_carriers_lohc: %w[
+      costs_carriers_lohc
+    ],
+    costs_carriers_ammonia: %w[
+      costs_carriers_ammonia
+    ],
+    costs_infrastructure_electricity: %w[
+      lv_net_costs_present
+      lv_net_costs_delta_present_future
+      lv_mv_trafo_costs_present
+      lv_mv_trafo_costs_delta_present_future
+      mv_net_costs_present
+      mv_net_costs_delta_present_future
+      mv_hv_trafo_costs_present
+      mv_hv_trafo_costs_delta_present_future
+      hv_net_costs_present
+      hv_net_costs_delta_present_future
+      interconnection_net_costs_present
+      interconnection_net_costs_delta_present_future
+      offshore_net_costs_present
+      offshore_net_costs_delta_present_future
     ],
-    costs_co2_energy_nodes: %w[
-      costs_molecule_graph
+    costs_infrastructure_heat: %w[
+      heat_infrastructure_ht_primary_pipelines
+      heat_infrastructure_ht_distribution_pipelines
+      heat_infrastructure_ht_distribution_stations
+      heat_infrastructure_ht_indoor
+      heat_infrastructure_mt_primary_pipelines
+      heat_infrastructure_mt_distribution_pipelines
+      heat_infrastructure_mt_distribution_stations
+      heat_infrastructure_mt_indoor
+      heat_infrastructure_lt_primary_pipelines
+      heat_infrastructure_lt_distribution_pipelines
+      heat_infrastructure_lt_distribution_stations
+      heat_infrastructure_lt_indoor
+    ],
+    costs_infrastructure_network_gas: %w[
+      gas_network_costs
     ]
   }.freeze
 
+  CSV_HEADER = [
+    'Group', 'Subgroup', 'Key',
+    'Total costs (eur)', 'Total CAPEX (eur)', 'Total OPEX (eur)',
+    'Capital costs (eur)', 'Depreciation costs (eur)',
+    'Fixed operation and maintenance costs (eur)', 'Variable operation and maintenance costs (eur)',
+    'Total investment over lifetime (eur)', 'WACC', 'Construction time (years)', 'Technical lifetime (years)',
+    'CO2 emission costs (eur)', 'Number of units'
+  ]
```

Comment:
If there are costs that specify "Total investment over lifetime (eur)", shouldn't all other costs then be specified as "(eur/y)"?